app/models/application_record.rb

```diff
@@ -68,27 +72,33 @@ def self.gender_eeo_data(start_date, end_date, department_ids)
                                                   .in_department(department_ids)
     all_genders = GENDER_OPTIONS | gender_records.pluck(:gender)
     all_genders.map do |option|
-      [option, gender_records.where(gender: option).count]
+      specific_records = gender_records.where(gender: option)
+      [option, specific_records.count, specific_records.interview_count]
     end
   end
 
+  # rubocop:disable Metrics/AbcSize
```

Comment:
That's a nifty line. Does this disable the Rubocop check for "too much stuff/logic in one method"?